.travis.yml

```diff
@@ -2,7 +2,6 @@ language: node_js
 before_script:
   - npm install -g npm
   - npm cache clear
-  - npm install -g broccoli-cli
```

Comment:
Yeah, I think gulp is preinstalled in Travis. Should I still have the line to be safe?